### ISSUE TITLE
Installation (lib/lib64)

### ISSUE THREAD

**Issue by irenekb** [ViennaRNA/RNAblueprint#3]
During installation two lib folders are created: lib and lib64
1) Installation Guide: Hint: in case of a local installation both must be added to the PYTHONPATH
2) Code update: merge them to only one lib

**Comment by j-4** [comment]
Hi,
its alright that lib and lib64 are created as we have architecture dependent binaries compiled here.
Concerning the documentation, feel free to send a RR where you add more details to the REAMDE section 
`In case of a local installation, please do not forget to adopt your path variables such as PATH, LD_LIBRARY_PATH, CPLUS_INCLUDE_PATH, PYTHONPATH, PERL5LIB`

**Comment by RaumZeit** [comment]
Hi Stefan,
I think the way the `swig` Python interface is installed currently still does not meet the standards. Of course, there are architecture dependent files that are installed, so everything needs to be installed to `$prefix/lib64` or similar, but virtually all Linux distributions set the configuration for Python such that it expects everything within this path anyway. The best thing one can do is to ask the Python interpreter itself where to install. While this was already done with the `m4` macro that come with `RNAblueprint`, the current situation still puts both, the `.py` and the `.so` into different places.

We had the same situation with the `ViennaRNA Package` before, and I was looking into how other packages do their installation of `swig` generated Python wrappers. In the end, I developed my own `m4` macros, which generate an actual Python package, with `__init__.py`, etc. which works very well in all environments we tested so far (incl. MacOS X)..

Long story short, I made a PR #4 that changes the autoconf/automake part of `RNAblueprint` to conform to the same standard we have for the `ViennaRNA Package`. Along with that, I also split the build process for Python 2.x and 3.x, such that both or just one of them are generated. Please have a look at my changes and consider merging.

Cheers,
Ronny

**Comment by j-4** [comment]
Thanks for the PR @RaumZeit ! I just released 1.3.1 and updated the conda package :-)
